docs/RelayAPI.md

```diff
@@ -1,376 +1,504 @@
-# RELAY API
+# trustlines network REST API
+The relay component of the trustlines network project provides a REST API.
 
-## Response Format
-Response format is based on some best practices. (see https://github.com/adnan-kamili/rest-api-response-format)
-
-## Base URL
-`/api/v1`
+## Base Endpoint
+```
+<protocol>://<host>:<port>/api/v1
+```
+You can use the following base endpoint to talk to the currently deployed KOVAN test setup.
+```
+https://relay0.testnet.trustlines.network:443/api/v1
+```
 
-## Error message
-In case of an error, the response of the relay API will have the following format:
+## Response
+- All endpoints return either a number, a JSON object or an array
```

Comment:
It's all JSON, a number is also various JSON. I would not write it that complicated. Just write that JSON is returned.